tendrl/alerting/handlers/__init__.py

```diff
@@ -88,6 +91,9 @@ def update_alert(self):
                 )
             )
 
+    def classify_alert(self):
```

Comment:
alert is a class object of each of the handlers. The NodeAlert is a subclass of Alert only so that the alert save path can be different. But other wise both are essentially same. So, when the handler manager says <specific_handler>.handle(...), alert is stored as an handle obj member (as in https://github.com/Tendrl/alerting/pull/39/files/7aff9978bf352022fbe621912b5c47feb0c159c0#diff-832c56a6c92c3a3783467abf5cef02c1R99) so classify alert can use self.alert so, no need to pass alert to it.

Also, alert.save() saves the alert in the generic path /alerting/alerts whereas, the classify_alert saves the alert in the node space(alerting/nodes/$Alert.node_id/$Alert.alert_id in etcd) or cluster space depending on what the alert corresponds to(This is similar to the way messages are stored not just in global context /Messages but also grouped under node and cluster namespaces/paths in etcd...) This enables fast filtering and other operations ...